src/api/hooks.ts

```diff
@@ -56,19 +63,21 @@ export const useAPI = <TResp, TError extends APIError = APIError>(
     error,
     {
       state: requestState,
+      statusCode: statusCode,
     },
   ];
 };
 
-type UseAPILazyHook<TResp, TArgs, TError extends APIError = APIError> = [
+export type UseAPILazyHook<TResp, TArgs, TError extends APIError = APIError> = [
   (args: TArgs) => Promise<APIResponse<TResp, TError>>,
   RequestInfo
 ];
 
 /**
  * Wrap a promise-based API call and return a callback to execute it. The callback returns an `APIResponse` so
  * success/failure can be determined at the call site.
- * @param fn An async function taking an object extending `SafeRequestArgs` and returning an `APIResponse<TResp, TError>`.
+ * @param fn An async function, similar to a useAPI argument. However, this function may take as an argument
+ * an object extending `SafeRequestArgs`.
```

Comment:
Should be `RequestArgs` instead of `SafeRequestArgs`

Reply:
https://github.com/UWmathNEWS/slugline-web/pull/33/commits/ab3ec32537484ef9cd0fb5f2d417549695f987ea